.snippets/code/Cargo.toml

```diff
@@ -1,35 +1,128 @@
 [workspace.package]
 license = "MIT-0"
-authors = ["Parity Technologies <admin@parity.io>", "Web3 Foundation", "PaperMoon.io"]
+authors = [
+  "Parity Technologies <admin@parity.io>",
+  "Web3 Foundation",
+  "PaperMoon.io",
+]
 homepage = "docs.polkadot.com"
 repository = "https://github.com/paritytech/polkadot-sdk-parachain-template.git"
 edition = "2021"
 
 [workspace]
 members = [
-    "tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet"
+  "tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet",
+  "tutorials/polkadot-sdk/parachains/zero-to-hero/pallet-unit-testing",
+  "tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime/runtime",
+  "tutorials/polkadot-sdk/parachains/zero-to-hero/pallet-benchmarking/runtime",
 ]
 resolver = "2"
 
 [workspace.dependencies]
+clap = { version = "4.5.10" }
 codec = { version = "3.6.12", default-features = false, package = "parity-scale-codec" }
-frame-support = { version = "38.0.0", default-features = false }
-frame-system = { version = "38.0.0", default-features = false }
-scale-info = { version = "2.11.1", default-features = false }
-sp-runtime = { version = "39.0.5", default-features = false }
+color-print = { version = "0.3.4" }
```

Comment:
FYI, if you use the umbrella crate here, you will have a much easier time updating dependencies.